price-feeder/oracle/oracle.go

```diff
@@ -31,7 +31,11 @@ import (
 // and broadcast pre-vote and vote transactions such that they're committed in
 // at least one block during each voting period.
 const (
-	tickerSleep = 1000 * time.Millisecond
+	tickerSleep = 1500 * time.Millisecond
```

Comment:
Let's not change this - we'd be using less node queries, but we'd miss more voting periods.
```suggestion
	tickerSleep = 1000 * time.Millisecond
```